components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -461,6 +467,64 @@ export function CreateWorkspacePage() {
     );
 }
 
+function LocalSSHWarning(props: { className: string }) {
+    const isLocalSSHProxyEnabled = useFeatureFlag("gitpod_desktop_use_local_ssh_proxy");
+    const [isFeedbackFormVisible, setFeedbackFormVisible] = useState<boolean>(false);
+
+    if (!isLocalSSHProxyEnabled) {
+        return null;
+    }
+
+    return (
+        <div className={props.className}>
+            <Alert
+                light
+                className="bg-gray-100 dark:bg-gray-800"
+                type="info"
+                iconSize="w-8 h-8"
+                icon={<Exclamation2 className="h-full w-full" />}
+                iconColor="text-gray-500"
+            >
+                To make VS Code Desktop open seamlessly we will add a single entry to your local SSH configuration.
+                Gitpod does not read any of your existing SSH configurations.&nbsp;
+                <a
+                    className="gp-link"
+                    href="https://www.gitpod.io/docs/references/ides-and-editors/vscode#modify-ssh-config-file"
+                    target="_blank"
+                    rel="noreferrer"
+                >
+                    More Detail
```

Comment:
**suggestion**: We've been using the _Learn more_ for all help links. Let's use that if we need this.

```suggestion
                    Learn more
```